wled00/bus_manager.h

```diff
@@ -361,6 +380,8 @@ struct BusConfig {
     if (start + count > total) total = start + count;
     return true;
   }
+
+  unsigned memUsage(unsigned nr = 0) const;
```

Comment:
`std::size_t` would be the more `std` way counting memory and stuff, but really, unsigned types and types of unspecified width should be avoided if possible